ckanext/switzerland/templates/scheming/display_snippets/qualified_relations.html

```diff
@@ -10,4 +11,14 @@
     {% endif %}
   {% endif %}
 {% endfor %}
+{% for see_also in see_alsos %}
+  {% if see_also and see_also.dataset_identifier %}
+    {% set related_dataset = h.get_dataset_by_identifier(see_also.dataset_identifier) %}
+    {% if related_dataset %}
+      <li>{{ h.link_to(related_dataset.title, h.url_for('dataset_read', id=related_dataset.name)) }}</li>
+    {% else %}
+      <li><a href="{{ see_also.relation }}">{{ see_also.dataset_identifier }}</a></li>
```

Comment:
should it be `see_also.dataset_identifier` or should we pointing to some `relation` ?

Reply:
thanks for catching that! it should be `dataset_identifier`. i will fix it.